css/modules.css

```diff
@@ -0,0 +1,130 @@
+.title {
+  font-size: 1.75em;
+  font-weight: bold;
+  float: left;
+}
+
+.icon-menu {
+  width: auto;
+  float: right;
+  font-size: 2em;
+}
+
+.menu {
+  display: none;
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  height: 100%;
+  width: 100%;
+  background-color: rgba(0,0,0,0.85);
+  z-index: 2;
+}
+
```

Comment:
This is where you should stick the `@media` query to style the `.menu` rather than in the states.css file.